### DISCUSSION TITLE
取消 _single_fligh插件是因为实际优化效果不佳么

### DISCUSSION THREAD

**Discussion by pmkol** [IrineSistiana/mosdns#399] [Q&A]
从插件的介绍看，感觉优化效果应该不错。取消掉这个插件，是因为ECS时，每个下游的请求实际不一样么，还是因为其它什么原因，觉得作者是比较注重性能优化的，所以很好奇原因，还麻烦作者帮忙解答下。

**Comment by IrineSistiana** [comment]
实际当然有效。然而没实际意义。

节省的cpu和流量忽略不计。

而且如果用了udp的上游还会有debuf，下游重传的包也会被滤去，稍微带点丢包率的网络会雪上加霜。
